src/throttler.guard.ts

```diff
@@ -112,4 +113,16 @@ export class ThrottlerGuard implements CanActivate {
     const prefix = `${context.getClass().name}-${context.getHandler().name}`;
     return md5(`${prefix}-${suffix}`);
   }
+
+  /**
+   * Throws an exception for the event that the rate limit has been exceeded.
+   *
+   * The context parameter allows to access the context when overwriting
+   * the method.
+   * @throws ThrottlerException
+   */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  protected throwThrottlingException(context: ExecutionContext): void {
```

Comment:
I think I like `throwException` better, but at that point it's just preference. Overall the change looks good. I'll try to set the rule up later to ignore `_` as the first character so we can remove the eslint comment